src/transformers/__init__.py

```diff
@@ -832,6 +834,15 @@
             "load_tf_weights_in_convbert",
         ]
     )
+    _import_structure["models.convnext"].extend(
+        [
+            "CONVNEXT_PRETRAINED_MODEL_ARCHIVE_LIST",
+            "ConvNextBlock",
```

Comment:
Not sure this one needs to be public, but it's your call.

Reply:
I will for now not include it in the init.